akka-sample-cqrs-java/src/main/resources/logback.xml

```diff
@@ -1,7 +1,7 @@
 <configuration>
     <appender name="STDOUT" target="System.out" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>[%date{ISO8601}] [%level] [%logger] [%marker] [%thread] - %msg {%mdc}%n</pattern>
+            <pattern>[%date{ISO8601}] [%level] [%logger] [{%mdc}] [%marker] [%thread] - %msg %n</pattern>
```

Comment:
`%mdc` can be rather much, so better to keep it at then end

Reply:
okay. I changed as for empty mdc it looks like an extra marker